test/test_poller_tag.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-2016: Alignak team, see AUTHORS.txt file for contributors
+#
+# This file is part of Alignak.
+#
+# Alignak is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Alignak is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Alignak.  If not, see <http://www.gnu.org/licenses/>.
+"""
+ This file is used to test poller tags
+"""
+
+import time
+from alignak.misc.serialization import unserialize
+
+from alignak_test import AlignakTest, unittest
+
+
+class TestPollerTag(AlignakTest):
+    """
+    This class tests the poller tag  of check
+    """
+    def setUp(self):
+        """
+        For each test load and check the configuration
+        :return: None
+        """
+        self.print_header()
+        self.setup_with_file('cfg/cfg_poller_tag.cfg')
+        self.assertTrue(self.conf_is_correct)
+
+        # Our scheduler
+        self._sched = self.schedulers['scheduler-master'].sched
+
+        # Our pollers
+        #print self._sched.pollers
+        #self._pollerm = self._sched.pollers['poller-master']
+        #self._pollern = self._sched.pollers['poller-north']
+        #self._pollers = self._sched.pollers['poller-south']
+
+        # No error messages
+        self.assertEqual(len(self.configuration_errors), 0)
+        # No warning messages
+        self.assertEqual(len(self.configuration_warnings), 0)
+
+    def test_poller_tag_command(self):
+        """
+        We have a command defined with poller_tag: north
+
+        :return:None
+        """
+        self.print_header()
+        host = self._sched.hosts.find_by_name("test_host_pt_01")
+        self.external_command_loop()
+        checks = self.schedulers['scheduler-master'].sched.checks.values()
+        mycheck = self._sched.checks[host.checks_in_progress[0]]
+        assert mycheck.poller_tag == 'north'
+
+    def test_poller_tag_host(self):
+        """
+        We have a host with a poller_tag: south
+
+        :return: None
+        """
+        self.print_header()
+        host = self._sched.hosts.find_by_name("test_host_pt_02")
+        self.external_command_loop()
+        checks = self.schedulers['scheduler-master'].sched.checks.values()
+        mycheck = self._sched.checks[host.checks_in_progress[0]]
+        assert mycheck.poller_tag == 'south'
+
+    def test_poller_tag_host_command(self):
+        """
+        We have a command with poller_tag: north
```

Comment:
Test description is better on the one and first line ... this for py.test to display a clean test name